dhis-2/dhis-services/dhis-service-tracker/src/test/resources/tracker/validations/enrollments_te_missing_ou.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
The idea of delete payload was to have the minimal information to in it
Something like `{
  "enrollments": [
    {
       "enrollment": "MNWZ6hnuhSw"
    }
  ]
}`

@stian-sandvold @Bekkalizer Can you confirm or disprove this?

Reply:
yes that could be the idea but the validation fails, so we need either to make it even with other strategies or modify the validation

Reply:
good point @enricocolasante glad you spotted this 😬

Reply:
Hey, the idea is that you just need the authority to delete, and delete something that exists without any unbreakable references to perform a delete. The payload @enricocolasante posted should be all you need, then all the other required info should be fetched before validation, and only the minimum should be validated.